MarvelCharsApp/MarvelCharsApp/Views/HomeViewController.swift

```diff
@@ -100,6 +123,24 @@ extension HomeViewController: CharacterPortraitDelegate {
     }
     
   }
-  
-  
+    
+}
+
+
+extension HomeViewController: UICollectionViewDelegate, UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return categoryButton.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "CategoryButtonCollectionViewCell", for: indexPath) as! CategoryButtonCollectionViewCell
```

Comment:
Identifiers and Nib names should go to the static class Resources. Never call directly with the String

Reply:
Also we avoid the use of force casting as!